frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/active-health-checks.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+import { MINUTE } from '../consts';
+
+describe('Test Popover behaviour for different active health check cases.', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    cy.install();
+    cy.visit('/ocs-dashboards/block-file');
+  });
+
+  after(() => {
+    checkErrors();
+    cy.logout();
+  });
+
+  const scaleDownAndOpenHC = (state: string, deployments: string[]) => {
+    const scaleTo = (replicas: number) =>
```

Comment:
Create a separate helper method `scaleDeployment(deployment: string, replicas: number)` for this ? Check if we already have one

Reply:
> Create a separate helper method scaleDeployment(deployment: string, replicas: number) for this ?

Sure.

> Check if we already have one

I looked into it. It seems we don't. See https://github.com/rexagod/console/blob/a462dbee4868dbfb5e94a92edcba0dad7433cf8d/frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts#L35.